README.md

```diff
@@ -1,216 +1,242 @@
 
-Now let's deploy Qlik Core and start to scale based on Qix active sessions.
+Now that the GKE cluster is set up and the documents are loaded into the cluster, you can deploy Qlik Core and start to scale based on Qlik Associative Engine active sessions.
 
-Start by adding ClusterRole for the Mira service
-```bash
-kubectl create -f ./rbac-config.yaml
-```
+1. Add a ClusterRole for the Mira service.
 
-Deploy Qlik Core:
-```bash
-kubectl create -f ./qlik-core
-```
+    ```bash
+    kubectl create -f ./rbac-config.yaml
+    ```
 
-The `engine` service exposes a custom metric named `qix_active_sessions`. 
+1. Deploy Qlik Core:
+
+    ```bash
+    kubectl create -f ./qlik-core
+    ```
+
+The `engine` service exposes a custom metric named `qix_active_sessions`.
 The Prometheus adapter removes the `_total` suffix and marks the metric as a counter metric.
 
-Get the total qix_active_sessions from the custom metrics API:
+1. Get the total Qlik Associative Engine active sessions from the custom metrics API:
 
-```bash
-kubectl get --raw "/apis/custom.metrics.k8s.io/v1beta1/namespaces/default/pods/*/qix_active_sessions" | jq .
-```
+    ```bash
+    kubectl get --raw "/apis/custom.metrics.k8s.io/v1beta1/namespaces/default/pods/*/qix_active_sessions" | jq .
+    ```
 
-Check that the HPA (Horizontal Pod Autoscaler), which is responsible for the scaling, is active and check that you have 0 sessions on your engines.
+1. Check that the Horizontal Pod Autoscaler (HPA), which is responsible for scaling, is active, and check that you have 0 sessions on your engines.
 
-```bash
-kubectl get hpa
-```
+    ```bash
+    kubectl get hpa
+    ```
 
 ## Monitor the cluster
-Before we start adding load to the cluster let's deploy Grafana so we can see how the cluster is reacting
 
-```bash
-kubectl create -f ./grafana
-```
+Before you add any load to the cluster, deploy Grafana for monitoring.
 
-Expose the grafana web server on a local port.
+1. Deploy Grafana.
 
-```bash
-./run.sh grafana
-```
+    ```bash
+    kubectl create -f ./grafana
+    ```
+
+1. Expose the Grafana web server on a local port.
+
+    ```bash
+    ./run.sh grafana
+    ```
 
-Now we can view grafana on http://localhost:3000
+You can view Grafana on http://localhost:3000.
 
 ## Add load to the cluster
 
-Now we can apply some load on the `engine` service with [core-qix-session-workout](https://github.com/qlik-oss/core-qix-session-workout)
+Now that Grafana is set up, apply some load on the `engine` service with [core-qix-session-workout](https://github.com/qlik-oss/core-qix-session-workout).
 
-First, we need to clone the repository and change folder to the repository directory.
+First, clone the repository and go to the repository directory.
 
-then we need to get the external ip-address from the nginx-controller which acts as the entrypoint to the cluster.
+Next, you need to get the external IP address from the nginx-controller which acts as the entry point to the cluster.
```

Comment:
I think entrypoint should be one word

Reply:
Entry point is usually two words whereas endpoint is usually one word.

I usually side with IBM globalization term base or MS terminology collection for tech terms.

https://www-01.ibm.com/software/globalization/terminology/e.html

https://www.microsoft.com/en-us/language/Search?&searchTerm=entry%20point&langID=Danish&Source=true&productid=All%20Products